### PR TITLE
02/10/24 button text fall back to value or

### LINKED ISSUE #28: if fields don't have a name they show undefined
IMAGE 2024-09-28 20:01:37

### PR DESCRIPTION
**Pull request by hardingjam**
## Motivation

Button text can be "undefined"

## Solution

❗ First, merge #110 

Fall back to the value property
If no value exists, then do not show a button
Closes #28 

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

**Review by thedavidmeister — CHANGES_REQUESTED** [review]
no tests or screenshots on this PR

**Comment by hardingjam** [comment]
image

image

File: __mocks__/mockFrameStates.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
+export const fieldsFrameState = {
```
Comment (thedavidmeister) [inline __mocks__/mockFrameStates.ts:1]: these mocks seem too big for the test

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

**Review by thedavidmeister — CHANGES_REQUESTED** [review]
why are we calling fixtures mocks?

**Review by thedavidmeister — APPROVED** [review]

**Pull request merged**